molecule/command/init/role.py

```diff
@@ -38,6 +38,16 @@ class Role(base.Base):
     .. option:: molecule init role --role-name foo
 
         Initialize a new role.
+
+    .. program:: molecule init role --role-name foo --template path
+
+    .. option:: molecule init role --role-name foo --template path
+
+        Initialize a new role using a local *cookiecutter* template. This allows
```

Comment:
How about:

> Initialize a new role using a local *cookiecutter* template. This allows the customization of a role while still using the default generated ``molecule`` folder. Please refer to the ``init scenario`` command in order to customise the default generated ``molecule`` folder.

Reply:
I changed it, but I still want to include the ansible-galaxy parallel. That was the whole point of this PR so I really want it to be explicit.